Don't print error and stack trace for the case of removing namespace's QOS limits if they have  not been set - log as a warning

y 06 18:01:22 ceph-nvme-vm8 bash[1656964]: [06-May-2024 15:01:22] INFO grpc.py:1654: Received request to delete namespace using NSID 401 from nqn.2016-06.io.spdk:cnode1, context: <grpc._server._Context object at 0x7fabc276ee80>, client address: IPv4 10.243.64.18:49600
May 06 18:01:22 ceph-nvme-vm8 bash[1656964]: [06-May-2024 15:01:22] INFO grpc.py:1176: Received request to remove namespace 401 from nqn.2016-06.io.spdk:cnode1, client address: IPv4 10.243.64.18:49600
May 06 18:01:22 ceph-nvme-vm8 bash[1656964]: [06-May-2024 15:01:22] ERROR grpc.py:1158: Error removing namespace's QOS limits, they might not have been set
May 06 18:01:22 ceph-nvme-vm8 bash[1656964]: Traceback (most recent call last):
May 06 18:01:22 ceph-nvme-vm8 bash[1656964]:   File "/src/control/grpc.py", line 1156, in remove_namespace_from_state
May 06 18:01:22 ceph-nvme-vm8 bash[1656964]:     self.gateway_state.remove_namespace_qos(nqn, str(nsid))
May 06 18:01:22 ceph-nvme-vm8 bash[1656964]:   File "/src/control/state.py", line 544, in remove_namespace_qos
May 06 18:01:22 ceph-nvme-vm8 bash[1656964]:     self.local.remove_namespace_qos(subsystem_nqn, nsid)
May 06 18:01:22 ceph-nvme-vm8 bash[1656964]:   File "/src/control/state.py", line 110, in remove_namespace_qos
May 06 18:01:22 ceph-nvme-vm8 bash[1656964]:     self._remove_key(key)
May 06 18:01:22 ceph-nvme-vm8 bash[1656964]:   File "/src/control/state.py", line 178, in _remove_key
May 06 18:01:22 ceph-nvme-vm8 bash[1656964]:     self.state.pop(key)
May 06 18:01:22 ceph-nvme-vm8 bash[1656964]: KeyError: 'qos_nqn.2016-06.io.spdk:cnode1_401'
May 06 18:01:22 ceph-nvme-vm8 bash[1656964]: [06-May-2024 15:01:22] INFO grpc.py:440: Received request to delete bdev bdev_7a11934c-bae0-4016-a46c-ba9aa6521ee2, client address: IPv4 10.243.64.18:49600

--- a/control/grpc.py
+++ b/control/grpc.py
@@ -1170,7 +1170,7 @@ class GatewayService(pb2_grpc.GatewayServicer):
         try:
             self.gateway_state.remove_namespace_qos(nqn, str(nsid))
         except Exception:
-            self.logger.exception(f"Error removing namespace's QOS limits, they might not have been set")
+            self.logger.warning(f"Error removing namespace's QOS limits, they might not have been set")
             pass
         try:
             self.gateway_state.remove_namespace(nqn, str(nsid))

